### PR TITLE
feature: allow comments in CMD_FLAGS.txt

### PR DESCRIPTION
**Pull request by deevis**
Allow comments in CMD_FLAGS.txt to allow things like this (blushing, yes this is my CMD_FLAGS.txt)

```
--api
--model=TheBloke_OpenOrca-Platypus2-13B-GPTQ_gptq-8bit-128g-actorder_True
--listen
--extensions dbh superbooga

# Other flags I use frequently
# ---------------------------------
# --trust-remote-code
#  --notebook 

# Models - the good, the bad, and the ugly
# -----------------------------------------------------------------------------
# Wow - good stuff - uncensored, pretty fast, good with patient evaluations...
# --model=TheBloke_MythoLogic-L2-13B-GPTQ_gptq-8bit-64g-actorder_True
# --model=TheBloke_vicuna-13B-v1.5-16K-GPTQ_gptq-8bit-128g-actorder_True
# --model=TheBloke_guanaco-13B-SuperHOT-8K-GPTQ' # needs trust_remote_code
# Uncensored AF - word association machine
# --model=TheBloke_Llama-2-13B-GPTQ_gptq-8bit-64g-actorder_True
```

### CODE REVIEW

File: webui.py [hunk 1]
```diff
@@ -17,7 +17,10 @@
     cmd_flags_path = os.path.join(script_dir, "CMD_FLAGS.txt")
     if os.path.exists(cmd_flags_path):
         with open(cmd_flags_path, 'r') as f:
-            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip())
+            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip() and not line.strip().startswith('#'))
+    if CMD_FLAGS:
```
Comment (oobabooga) [inline webui.py:21]: CMD_FLAGS may not be defined in this line. I think it's fine to remove this print statement as a whole

Comment (deevis) [reply webui.py]: Good catch - thank you.

I've fixed the error by moving the output to live within the `os.path.exists` block, ensuring CMD_FLAGS is set.

I find that having the flags displayed upon startup serves as a bit of a reassuring sanity check.  I'm happy to remove it, though, if you find it to be more noise than signal ;)

Comment (deevis) [reply webui.py]: FYI - I've tested the following cases with this PR locally:

CMD_FLAGS.txt
  1.  does not exist
  2.  is empty
  3.  has flags but no commented lines
  4.  has flags and commented lines

File: webui.py [hunk 2]
```diff
@@ -17,7 +17,10 @@
     cmd_flags_path = os.path.join(script_dir, "CMD_FLAGS.txt")
     if os.path.exists(cmd_flags_path):
         with open(cmd_flags_path, 'r') as f:
-            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip())
+            CMD_FLAGS = ' '.join(line.strip() for line in f.read().splitlines() if line.strip() and not line.strip().startswith('#'))
```
Comment (missionfloyd) [inline webui.py:20]:
```suggestion
            CMD_FLAGS = ' '.join(line.strip() for line in f if line.strip() and not line.strip().startswith('#'))
```
You don't need `read()` to iterate lines in a file.

Comment (deevis) [reply webui.py]: Thanks @missionfloyd - that's better now

**Pull request merged**